package.json

```diff
@@ -57,8 +56,8 @@
     "eslint-plugin-react": "^7.22.0",
     "isomorphic-unfetch": "^3.1.0",
     "jest": "^26.6.3",
-    "postcss-import": "^14.0.0",
+    "scheduler": "^0.20.2",
```

Comment:
Is this dependency necessary?

Reply:
Yep: https://github.com/preactjs/next-plugin-preact/issues/49